### ISSUE TITLE
[TISMIR] Missing section numbers for titlesec 2.10.1 users

### ISSUE THREAD

**Issue by franrodalg** [ismir/paper_templates#8]
Some authors reported that pdfs compiled in their machines missed the section numbers. This seems to be a bug in titlesec package 2.10.1, which apparently is the version included in Ubuntu 16.04 (and maybe other distributions as well). 

There is a solution available [online](https://bugs.launchpad.net/ubuntu/+source/texlive-extra/+bug/1574052) for this issue. Add:
```
\usepackage{etoolbox}

\makeatletter
\patchcmd{\ttlh@hang}{\parindent\z@}{\parindent\z@\leavevmode}{}{}
\patchcmd{\ttlh@hang}{\noindent}{}{}{}
\makeatother
 ```
after `\usepackage{titlesec}` in `tismir.sty`. 

Pull request #9 should fix this.

**Comment by stefan-balke** [comment]
Sorry for the delay here @franrodalg.
Anyone still on 16.04 with that `titlesec` version and could test this? @fzalkow maybe (sorry :))?

Is this compatible with newer `titlesec` versions?

**Comment by fzalkow** [comment]
I am on Ununtu 16.04, but using this [PPA](https://launchpad.net/~jonathonf/+archive/ubuntu/texlive-2018) for having texlive 2018. Cannot confirm this bug with texlive 2018 (having titlesec 2.10.2).

However, I could confirm the bug as well as the fix using the docker image `raabf/latex-versions:texlive2015` (Ubuntu 16.04, texlive 2015, titlesec 2.10.1).

Edit: The fix brings no problems with titlesec 2.10.2 as far as I see.

**Comment by stefan-balke** [comment]
Okay, thank you very much, @fzalkow.

@franrodalg could you rebase on the current master? Then I'll merge!

**Comment by franrodalg** [comment]
Hi @stefan-balke. I might be making a noob mistake, as it's been a while, but rebasing doesn't seem to do anything. I'm currently following the instructions in [here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request), as the official [Github instructions](https://help.github.com/articles/about-pull-request-merges/) don't seem very detailed. Should I be pulling instead of fetching changes in the repo?

**Comment by stefan-balke** [comment]
Yeah, try pulling. The changes shouldn't affect the TISMIR stuff anyways...

**Comment by f0k** [comment]
> Yeah, try pulling.

Usually, you should be fetching. You'll need both your fork and the repository you want to send the PR to as git remotes (often called `origin` and `upstream`). Then you'd do `git fetch upstream` and, in your branch, `git rebase upstream/master`. Afterwards you'd force-push with `git push --force`.

> The changes shouldn't affect the TISMIR stuff anyways...

So actually there's no need to rebase! Otherwise github would note that there's a conflict.

**Comment by franrodalg** [comment]
Tans @f0k ! That was really helpful! It had been so long that I had completely forgotten I didn't have this repo as remote. Silly me

 Should be done now :)

**Comment by stefan-balke** [comment]
Ooops, yes, my bad. Since no conflicts exists, this is a simple merge...thank!

**Comment by stefan-balke** [comment]
Solved in PR #9
